yaml/roads.yaml

```diff
@@ -32,6 +31,14 @@ global:
       sport: {col: sport}
       man_made: {col: man_made}
       walking_network: {expr: "mz_hiking_network(osm_id)"}
+      cycle_network: {expr: "mz_cycling_network(tags, osm_id)"}
```

Comment:
Nitpick: I think there was some discussion about making it `bicycle_network` rather than `cycle_network`, for consistency or something?